src/cloud/components/molecules/Editor/index.tsx

```diff
@@ -637,25 +636,22 @@ const Editor = ({
     setScrollSync(not)
   }, [])
 
-  const embeddableDocs = useMemo(() => {
-    const embedMap = new Map<string, EmbedDoc>()
-    if (team == null) {
-      return embedMap
-    }
-
-    for (const doc of docsMap.values()) {
-      if (doc.head != null) {
-        const current = `${location.protocol}//${location.host}`
-        const link = `${current}${getTeamURL(team)}${getDocURL(doc)}`
-        embedMap.set(doc.id, {
-          title: doc.title,
-          content: doc.head.content,
-          link,
-        })
+  const getEmbed = useCallback(
+    async (id: string) => {
+      const doc = await loadDoc(id)
+      if (doc == null || team == null) {
+        return undefined
       }
-    }
-    return embedMap
-  }, [docsMap, team])
+      const current = `${location.protocol}//${location.host}`
+      const link = `${current}${getTeamURL(team)}${getDocURL(doc)}`
```

Comment:
Could be shortened to |
` const link = `${current}${getDocLinkHref(doc, team)}``